src/tbp/monty/frameworks/models/feature_location_matching.py

```diff
@@ -160,12 +160,12 @@ def receive_votes(self, vote_data):
                 # Check that object is still in matches after ID update
                 if possible_obj in self.possible_matches:
                     # TODO: better way to dynamically adapt k
-                    if vote_data["pos_location_votes"][possible_obj].shape[0] < 5:
+                    if vote_data["pos_location_votes"][possible_obj].shape[0] < 4:
```

Comment:
Normally I would say that this "[magic constant](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)" 4 should be replaced with an actual constant, especially since it's used again below and without it, it's not obvious that the two 4s are related. 

But since this is a temporary fix, it's probably fine.

Reply:
that's a good point. I updated it